### DISCUSSION TITLE
Gracefully ignore missing folders

### DISCUSSION THREAD

**Discussion by stuartcusackie** [spatie/laravel-backup#1671] [Ideas]
I am including folders that don't exist yet but they could be created at any point by user interactions on my application (It's a Statamic website):

Example:
```
'include' => [
      ...
      resource_path('forms'),
      ....
],
```

The 'forms' folder does not exist but it is likely to be created at some point as form submissions come through. There are other folders like this. Also, when deploying through GitHub, empty folders are not created on the server (Forge). So although a folder might exist locally. It won't be created on the server on deployment if it is empty.

At the moment the whole backup process fails if the folder does not exist:
`Backup failed because The "/home/forge/mywebsite.ie/resources/forms" directory does not exist..`
Also, no notifications are sent about this. You have to check the error logs.

I tried setting this variable to true but it doesn't seem to apply to missing directories:
`'ignore_unreadable_directories' => true,`

I have two choices at the moment, neither of which are ideal:
- Remove the empty folder from the 'include' list (I'll have to remember to re-add it in the future).
- Create the folder manually on the server.

**Can the backup process be updated to gracefully ignore missing folders?**

Thanks. I love this package!

**Comment by parallels999** [comment]
Feel free to make a PR
